source/main/webpack.config.js

```diff
@@ -4,7 +4,7 @@ const yamljs = require('yamljs');
 
 let reportUrl = '';
 try {
-  reportUrl = yamljs.parseFile('installers/launcher-config-windows.yaml').reportServer;
+  reportUrl = yamljs.parseFile('installers/launcher-config.yaml').reportServer;
 } catch (e) {} // eslint-disable-line
```

Comment:
we should also remove this try-catch block and fail if the config cant be found
the silent error has caused the linux beta to use staging report server

Reply:
Done.